lib/emscripten/src/lib.rs

```diff
@@ -265,7 +222,7 @@ pub struct EmscriptenGlobals {
 }
 
 impl EmscriptenGlobals {
-    pub fn new(module: &Module) -> Self {
+    pub fn new(module: &Module /*, static_bump: u32 */) -> Self {
```

Comment:
Is this needed `/*, static_bump: u32 */` ?

Reply:
the `static_bump` is emscripten module independent, but we've hardcoded the one for nginx here. The user has to pull the constant out of the generated js file, since there is no provided metadata in emscripten modules. We should make it variable at some point, and this is a reminder.